workspaces/getting-started.md

```diff
@@ -3,6 +3,11 @@ title: "Getting started"
 description: "Learn how to develop in a workspace."
 ---
 
+This article walks you through creating a new workspace using a pre-defined
+image.
+
+![Relationship between images and workspaces](../assets/workspaces/lifecycles.png)
```

Comment:
This image looks pretty complex and I'm not sure what we're trying to get at here... @mterhar I think this was originally a sketch you put together, right? Can we add some explanatory text that describes what we're trying to demonstrate with this image?

Reply:
I don't know if this is the greatest space for it -- I just wanted to try it out and see what y'all think. I don't know if we would be better off turning this visual into a guide on how Coder works under-the-hood, so to speak?

Reply:
This was originally a diagram I created for a blog post. Will defer to y'all but I think this would be well suited for the workspaces/index and when referring to workspace templates :)